src/main/java/com/Proiect_Varianta_Buna/demo/Services/EventService.java

```diff
@@ -0,0 +1,25 @@
+package com.Proiect_Varianta_Buna.demo.Services;
+
+import com.Proiect_Varianta_Buna.demo.Interfaces.IEventService;
+import com.Proiect_Varianta_Buna.demo.Repositories.EventRepository;
+import com.Proiect_Varianta_Buna.demo.TableEntities.Event;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+public class EventService implements IEventService {
+
+    private final EventRepository eventRepository;
+
+    public EventService(EventRepository eventRepository) {
+        this.eventRepository = eventRepository;
+    }
+
+
+
+    @Override
+    public List<Event> getEvents() {
+        return eventRepository.findAll();
```

Comment:
Te rog sa creezi clase de mapper dupa ce creezi DTOs si sa le folosesti in Implementarea Serviciilor pentru a mapa clasele de Entities la cele de DTOs 

Exemplu bun poti sa gasesti pe acest PR: https://github.com/MateiPop12/en-summer-apprentice-java/pull/1